src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -111,6 +111,27 @@ def migrate(self, daemon_name=None, realm_name=None, zonegroup_name=None, zone_n
                                                          secret_key)
         return result
 
+    @RESTController.Collection(method='PUT', path='/setup_multisite_replication')
+    @allow_empty_body
+    # pylint: disable=W0102,W0613
+    def setup_multisite_replication(self, daemon_name=None, realm_name=None, zonegroup_name=None,
+                                    zonegroup_endpoints=None, zone_name=None, zone_endpoints=None,
+                                    username=None, cluster=None):
```

Comment:
By cluster you mean fsid of the cluster?
```suggestion
    def setup_multisite_replication(self, daemon_name=None, realm_name=None, zonegroup_name=None,
                                    zonegroup_endpoints=None, zone_name=None, zone_endpoints=None,
                                    username=None, cluster_fsid=None):
```